### ISSUE TITLE
QVariantMap : Switch account / uid [ Metab community ]

### ISSUE THREAD

**Issue by sbDevGit** [ucoin-io/cutecoin#252]
When I wanted to switch account 
s_b_e_ ( not member currently ) to 
sbdev ( not member & has ever been )
```bash
sb@sb:~/mesProgrammes/uCoinApps/CuteCoin$ ./cutecoin 
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
uint DBusMenuExporterDBus::GetLayout(int, int, const QStringList&, DBusMenuLayoutItem&): Condition failed: menu 
uint DBusMenuExporterDBus::GetLayout(int, int, const QStringList&, DBusMenuLayoutItem&): Condition failed: menu 
Traceback (most recent call last):
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 210, in watch
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 223, in start_perpetual_crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 285, in crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 368, in peering_traversal
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 270, in refresh_state
TypeError: string indices must be integers
Traceback (most recent call last):
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 210, in watch
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 223, in start_perpetual_crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 285, in crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 368, in peering_traversal
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 269, in refresh_state
  File "/home/gael/cutecoin/lib/ucoinpy/api/bma/__init__.py", line 140, in merkle_easy_parser
  File "/home/gael/cutecoin/lib/ucoinpy/api/bma/__init__.py", line 116, in requests_get
ValueError: status code != 200 => 502 (<html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.7.2</center>
</body>
</html>
)
Traceback (most recent call last):
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 210, in watch
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 223, in start_perpetual_crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/network.py", line 285, in crawling
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 368, in peering_traversal
  File "/home/gael/cutecoin/src/cutecoin/core/net/node.py", line 269, in refresh_state
  File "/home/gael/cutecoin/lib/ucoinpy/api/bma/__init__.py", line 140, in merkle_easy_parser
  File "/home/gael/cutecoin/lib/ucoinpy/api/bma/__init__.py", line 116, in requests_get
ValueError: status code != 200 => 502 (<html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.7.2</center>
</body>
</html>
)
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
QVariantMap DBusMenuExporterDBus::getProperties(int, const QStringList&) const: Condition failed: action 
Erreur de segmentation (core dumped)
```

**Comment by Insoleet** [comment]
A node answered incorrectly and crashed cutecoin. Thanks for the catch.

**Comment by sbDevGit** [comment]
the 10.2 :)
thanks !
by the way I just remember than my uid s_b_e_ is revoked , to help to target the bug

**Comment by Insoleet** [comment]
Well I won't fix it since we are really near a 0.11 release. You can try to build it if you want. If you are running Windows, there are automatic windows builds available on appveyor  : https://ci.appveyor.com/project/Insoleet/cutecoin/history

Select a build, select an environment instance, go to "Artifact" tab.
